deployments/azure-functions-app-container.yml

```diff
@@ -0,0 +1,57 @@
+# Action Requires
+# 1. Setup the AZURE_CREDENTIALS secrets in your GitHub Repository
+# 2. Setup the REGISTRY_USERNAME secrets in your GitHub Repository
+# 3. Setup the REGISTRY_PASSWORD secrets in your GitHub Repository
+# 4. Replace REGISTRY, NAMESPACE, IMAGE, TAG in the following template with proper values
+# 5. Add this yaml file to your project's .github/workflows/
+# 6. Push your local project to your GitHub Repository
```

Comment:
These two sentences are redundant here as the Actions UI will guide user on that and need not be done separately.